### DISCUSSION TITLE
Running SDDM on wayland with labwc

### DISCUSSION THREAD

**Discussion by stefonarch** [labwc/labwc#1860] [General]
Documentation is not at the top, and nowhere I found hints about theme incompatibly. Working are no theme and "breeze" while my default theme produced a black window with labwc menu to exit.

Anyway, I managed to run it now:

`/etc/sddm.conf `
```

[General]
...
DisplayServer=wayland
GreeterEnvironment=QT_WAYLAND_SHELL_INTEGRATION=layer-shell

 [Wayland]
...
CompositorCommand=labwc
        
```
The best thing I found [here](https://bbs.archlinux.org/viewtopic.php?id=289612) :
It's possible to configure cursor size, shape and speed using `/var/lib/sddm/.config/labwc`.

>SDDM runs the greeter as a system user named "sddm" whose home directory needs to be set to /var/lib/sddm.

**Comment by 01micko** [comment]
While not _strictly_ on topic I am using [nwg-hello](https://github.com/nwg-piotr/nwg-hello) by @nwg-piotr for my DM in debian which uses [greetd](https://git.sr.ht/~kennylevinsen/greetd) by @kennylevinsen

I managed to persuade Piotr to put the instuctions for labwc in the [nwg-hello/README.md](https://github.com/nwg-piotr/nwg-hello/blob/main/README.md#running-on-debian-and-labwc) though https://github.com/nwg-piotr/nwg-hello/issues/16 so it is documented. There was absolutely nothing I found to use `labwc` with `greetd` so I started with baby steps using `wlgreet` and that worked so progressed to `nwg-hello`. I even put in a basic `rc.xml` to take a screen-shot with `grim` that is saved in the greeter's home! (`/etc/greetd`).
